### PR TITLE
[T] (quests) simple_storage

### PR DESCRIPTION
**Pull request by luo-2233**
Co-authored-by: 渊荀 <1046567763@qq.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/simple_storage.snbt [hunk 1]
```diff
@@ -163,7 +163,7 @@
 		{
 			x: 1.5d
 			y: 1.0d
-			description: ["Transfer larger volumes of items at a time."]
+			description: ["一次传输大量项目。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/simple_storage.snbt]:
```suggestion
			description: ["一次传输大量物品。"]
```

File: config/ftbquests/quests/chapters/simple_storage.snbt [hunk 2]
```diff
@@ -281,7 +281,7 @@
 		{
 			x: 6.5d
 			y: 1.0d
-			description: ["Set the stack size to one!"]
+			description: ["将堆栈大小设置为 1！"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/simple_storage.snbt]:
```suggestion
			description: ["将堆叠大小设置为1。"]
```

File: config/ftbquests/quests/chapters/simple_storage.snbt [hunk 3]
```diff
@@ -257,7 +257,7 @@
 			y: 1.0d
 			description: [
 				""
-				"Processes the specific amount of items in the filter."
+				"处理筛选器中的特定数量的项目。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/simple_storage.snbt]:
```suggestion
				"处理过滤器中的特定数量的物品。"
```

**Review by Cactusstudent — CHANGES_REQUESTED** [review]
Machine Translation

**Review by AnzhiZhang — APPROVED** [review]

**Review by Cactusstudent — APPROVED** [review]

**Pull request merged**